frontend/src/components/Notification.tsx

```diff
@@ -0,0 +1,47 @@
+import { Snackbar, makeStyles } from '@material-ui/core'
+import * as React from 'react';
+import Alert from '@material-ui/lab/Alert';
+
+
+type NotificationType =  {
+    isOpen: boolean;
+    type: 'error' | 'info' | 'success' | 'warning'; 
+    message: string;
+    setisOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    handleClose?: any
+};
+
+const useStyles = makeStyles(theme => ({
+    root:{
+        top: theme.spacing(9) //Change this depending on size of header
+    }
+}));
+
+export default function Notification(props: NotificationType) {
+    const classes = useStyles()
+
+    const handleClose = (e) => { 
+        props.setisOpen(false)
```

Comment:
Make this prop part of this component's state so the parent doesn't have to worry about managing it